tests/ert_tests/ert3/data/test_transmitters.py

```diff
@@ -35,16 +42,31 @@ def shared_disk_factory(name: str) -> SharedDiskRecordTransmitter:
 
 
 @contextlib.contextmanager
-def in_memory_factory_context():
+def in_memory_factory_context(*args):
```

Comment:
Should we make these `kwargs` instead, that way the context is not dependent on the order of the arguments?